tests/Macros/PaginateTest.php

```diff
@@ -1,11 +1,10 @@
 <?php
 
-namespace Spatie\CollectionMacros\Test\Macros;
+namespace Spatie\CollectionMacros\Test;
 
 use Illuminate\Support\Collection;
```

Comment:
As a follow-up to the comment above, we now need to import the base `TestCase`-class:
```suggestion
use Illuminate\Support\Collection;
use Spatie\CollectionMacros\Test;
```

Reply:
good point! It should be `use Spatie\CollectionMacros\Test\TestCase;` though. There are a few tests have the same problem, wondering if I should fix it in this PR a create a separate one...